src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVFileUtils.java

```diff
@@ -108,35 +98,37 @@ private static SAMFileWriter getSAMFileWriter(final String outputName, final SAM
             }
         }
         catch ( final IOException ioe ) {
-            throw new GATKException("Unable to read kmers from "+kmersFile, ioe);
+            throw new GATKException("Unable to read kmers from "+kmersFilePath, ioe);
```

Comment:
Spaces around "+", just bother if you are going to change this file for any other reason , not for this alone.

Reply:
done